tests/registration/registration.spec.js

```diff
@@ -0,0 +1,303 @@
+      "button.btn.btn-link:nth-child(1)"
+    );
+    await registrationButton.click();
+
+    const popupRegistration = page.locator("div.modal-content");
+    await expect(
+      popupRegistration,
+      "Registration popup should be visible"
+    ).toBeVisible();
+
+    const nameInput = popupRegistration.locator("input#signupName");
+    const lastNameInput = popupRegistration.locator("input#signupLastName");
+    const emailInput = popupRegistration.locator("input#signupEmail");
+    const passwordInput = popupRegistration.locator("input#signupPassword");
+    const repeatPasswordInput = popupRegistration.locator(
+      "input#signupRepeatPassword"
+    );
+    const registerButton = popupRegistration.locator(
+      "button.btn.btn-primary:nth-child(1)"
+    );
+
+    await nameInput.fill(name);
+    await lastNameInput.fill(lastName);
+    await emailInput.fill(email);
+    await passwordInput.fill(password);
+    await repeatPasswordInput.fill(repeatPassword);
+    await registerButton.click();
+
+    await expect(page).toHaveURL("https://qauto.forstudy.space/panel/garage");
+
+    const myProfileDropDown = page.locator("button#userNavDropdown");
+    await myProfileDropDown.click();
+
+    const profileOption = page.locator(
+      "div.user-nav_menu-group a:nth-child(2)"
+    );
+    await profileOption.click();
+
+    const userInfo = page.locator("p.profile_name.display-4");
+    expect(await userInfo.innerText()).toEqual(name + " " + lastName);
+  });
+
+  // #2 Fields are mandatory
+  test("Fields are mandatory", async ({ page }) => {
+    await page.goto("/");
+    const signinButton = page.locator(
+      "button.btn.btn-outline-white.header_signin"
+    );
+    await signinButton.click();
+
+    const popupLogin = page.locator("div.modal-content");
+    await expect(popupLogin, "Log in popup should be visible").toBeVisible();
+
+    const registrationButton = popupLogin.locator(
+      "button.btn.btn-link:nth-child(1)"
+    );
+    await registrationButton.click();
+
+    const popupRegistration = page.locator("div.modal-content");
+    await expect(
+      popupRegistration,
+      "Registration popup should be visible"
+    ).toBeVisible();
+
+    const nameInput = popupRegistration.locator("input#signupName");
+    const lastNameInput = popupRegistration.locator("input#signupLastName");
+    const emailInput = popupRegistration.locator("input#signupEmail");
+    const passwordInput = popupRegistration.locator("input#signupPassword");
+    const repeatPasswordInput = popupRegistration.locator(
+      "input#signupRepeatPassword"
+    );
+
+    await nameInput.click();
+    await lastNameInput.click();
+    await emailInput.click();
+    await passwordInput.click();
+    await repeatPasswordInput.click();
+    await passwordInput.click();
+
+    let errorLabel;
+    errorLabel = popupRegistration.locator("form > div:nth-child(1) > div > p");
+    expect(await errorLabel.innerText()).toEqual(errorNameRequiredText);
```

Comment:
replace all innerText() calls with web first assertions
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text